wolfcrypt/benchmark/benchmark.c

```diff
@@ -1695,15 +1774,40 @@ static void bench_stats_sym_finish(const char* desc, int useDeviceID, int count,
     defined(HAVE_CURVE25519) || defined(HAVE_ED25519) || \
     defined(HAVE_CURVE448) || defined(HAVE_ED448) || \
     defined(WOLFSSL_HAVE_KYBER)
-static void bench_stats_asym_finish(const char* algo, int strength,
-    const char* desc, int useDeviceID, int count, double start, int ret)
+static void bench_stats_asym_finish_ex(const char* algo, int strength,
+    const char* desc, const char* desc_extra, int useDeviceID, int count,
+    double start, int ret)
 {
     double total, each = 0, opsSec, milliEach;
     const char **word = bench_result_words2[lng_index];
     const char* kOpsSec = "Ops/Sec";
-    char msg[128] = {0};
+    char msg[256] = {0};
+    static int asym_header_printed = 0;
 
     total = current_time(0) - start;
+
+#ifdef LINUX_RUSAGE_UTIME
```

Comment:
Is there any place this code can execute once and not be duplicated several times?

Reply:
good idea -- and moreover, I had double-declared static vars `base_rusage` and `cur_rusage`.

Reply:
oh and, the code is _supposed_ to be executed once for each subtest, to localize where the syscall overhead is running away.  but anyway the main thing was that the duplicated logic didn't need to be duplicated.  fixed that as you saw.